benefits/static/css/styles.css

```diff
@@ -163,6 +167,16 @@ footer {
   margin-bottom: 2rem;
 }
 
+.container.content h1 {
+  font-size: 24px;
+  line-height: 30px;
+}
+
+.container.content p {
+  font-size: 18px;
+  line-height: 27px;
```

Comment:
I think these rules belong in the [`p` block](https://github.com/cal-itp/benefits/blob/dev/benefits/static/css/styles.css#L41) up at the top of the document

Reply:
Also,  letter spacing is now 0.05em: `letter-spacing: 0.05em;`